06_PDDE_Checker.R

```diff
@@ -227,10 +234,93 @@ zero_utilization <- Project0 %>%
   mutate(Detail = "") %>%
   select(all_of(PDDEcols))
 
-##### For later-------
-# Incompatible Funding Source and Project Type Funding Source X can only be used with Project Type Y. Project Type A can only be used with Funding Source B. (this will need a lot more detail, hold on this one)
+# For later.. -------------------------------------------------------------
+
+# Incompatible Funding Source and Project Type Funding Source X can only be used
+# with Project Type Y. Project Type A can only be used with Funding Source B.
+# (this will need a lot more detail, hold on this one)
+
+
+# Overlapping participations ----------------------------------------------
```

Comment:
ideally the two dataframes below would be "funcion"ified, since they're so similar, but seems good to me!